packages/ui-progress/src/ProgressCircle/index.tsx

```diff
@@ -65,20 +65,18 @@ class ProgressCircle extends Component<
     shouldAnimateOnMount: false,
 
     // default to showing `success` color on completion
-    // @ts-expect-error ts-migrate(7031) FIXME: Binding element 'valueNow' implicitly has an 'any'... Remove this comment to see the full error message
-    meterColor: ({ valueNow, valueMax }) =>
+    meterColor: ({ valueNow, valueMax }: Values) =>
       valueNow / valueMax >= 1 ? 'success' : 'brand'
   }
 
-  _timeouts = []
+  _timeouts: NodeJS.Timeout[] = []
```

Comment:
Just a small note:
I know that for some reason TypeScript thinks this is a Node package (probably because the packages.json's `type` field is set to `commonjs`) and thus giving back Node.js related types for apis such as: `setTimeout`, but this code will not run in a Node environment, so this type is not correct here.

A platform independent way of doing the typing for such apis is this:

```typescript

  _timeouts: Array<ReturnType<typeof setTimeout>> = []

```

Reply:
Another note:
this will still set the underlying type to `NodeJS.Timeout`, since TS thinks this is a Node environment, so we should be careful with these kind of situations, and have to remember to not use any kind of Node functionality. (even though the type would allow such things, but the runtime for it is simply not there)

Reply:
Interesting notes, thanks for explaining it to me (: